### ISSUE TITLE
[I18N] Fix all instances of incorrect translation syntax

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/silverstripe-framework#8356]
## Affected Version

SilverStripe 4.x

## Description

When running the text collector there are a lot of strings that throw errors and are ignored. We have an [open issue to support `SomeClass::class` syntax in translations](https://github.com/silverstripe/silverstripe-framework/issues/7647), but until that is completed we should fix the existing translations.

## Steps to Reproduce

* Run `vendor/bin/sake dev/tasks/i18nTextCollectorTask '' flush=1`
* See a tonne of errors
* Check and notice that the examples that cause errors don't make it into `framework/lang/en.yml` as source strings

## Pull requests

- [ ] #8357
- [ ] https://github.com/silverstripe/silverstripe-siteconfig/pull/100
- [ ] https://github.com/silverstripe/silverstripe-versioned-admin/pull/58

**Comment by robbieaverill** [comment]
PRs added that fix a relatively vanilla 4.3 install. I'll sanity check the wider CWP modules too.

**Comment by robbieaverill** [comment]
Ok follow up PRs for CWP and Fluent are up.

It'd be nice to display translation errors in a nice way rather than having a screen full of exception stack traces, and also to exit with a non-zero exit code when they happen so that we could put this into CI. I had a brief look at both of these. We can only really get the warning message e.g. "missing translation key for X" but in doing this we lose all of the stack trace context, which is actually more useful in this task. Re: exiting with non-zero exit codes, we can do this too but we also lose the exception context by defining a new error handler.

TL;DR: wait until we have a proper CLI in place, then we can redesign text collection generally. Related: https://github.com/silverstripe/silverstripe-framework/issues/5542

**Comment by kinglozzer** [comment]
PRs all merged, closing as root cause is tracked in https://github.com/silverstripe/silverstripe-framework/issues/7647
